[fix]remove login_email from okta idp custom claims (#4175)
Kong/docs.konghq.com: app/konnect/org-management/okta-idp.md

> 1. Go to the Claims tab.

Reviewer: ```suggestion
1. Go to the Claims tab to configure the `groups` claim.
```

>     Configure the claim: `groups`.

Reviewer: ```suggestion
```
I was working with this stuff yesterday and `groups` is something you specify, so it's not the actual name of something in the UI.

    re-create them directly in Okta. They can do this by exporting the group in
    question in CSV format, then importing the CSV file to populate the new group.

1. Select **Create** to save.

> If you have problems setting up these claims, refer to the Okta documentation

Reviewer: ```suggestion
If you have problems setting up this claim, refer to the Okta documentation
```
